server/settings/templated_workfile_build.py

```diff
@@ -28,6 +28,7 @@ class TemplatedWorkfileProfileModel(BaseSettingsModel):
 
 
 class TemplatedWorkfileBuildModel(BaseSettingsModel):
+    """More complex workfile template builder with Placeholders"""
```

Comment:
I wouldn't name this "More complex" - that has a way too negative connotation that way.

@m-u-r-p-h-y @philippe-ynput thoughts?